app/views/admin/friends/activities/_activity.html.erb

```diff
@@ -3,6 +3,9 @@
     <strong>
       <%= link_to "#{activity.event.humanize}", edit_community_admin_friend_activity_path(current_community.slug, friend, activity), remote: true, class: 'open_activity_modal' %>
       <%= activity.occur_at.strftime("-- %I:%M %p, %A, %B %-d, %Y") %>
+      <% if activity.accompaniable? %>
+        <em>3 volunteers needed.</em>
```

Comment:
Nice catch on adding this to the admin template, I didn't think of that!  Was thinking we could reuse the activity.volunteers_needed method here so that admins are seeing a consistent message with volunteers/accompaniment_leaders.

Reply:
oops! ha! that was definitely an oversight!